src/Controller/CRUDController.php

```diff
@@ -61,10 +73,60 @@ public function setContainer(ContainerInterface $container = null)
         $this->configure();
     }
 
+    // BC for Symfony 3.3 where ControllerTrait exists but does not contain get() and has() methods.
+    public function __call($method, $arguments)
```

Comment:
You will have to replace `is_callable` with `method_exists` somewhere in the code to fix the tests, look for it

Reply:
Uuuh wait nevermind, that was in case `__call` was used to replace `render` Travis passes.

Reply:
Yes, I "borrowed" some parts from your PR, including that one. See the new line 522:
https://github.com/sonata-project/SonataAdminBundle/pull/4785/files#diff-a8a33ea35d3afb5f4ce4a69f18516b03R522

Reply:
Oh right didn't see... and it is needed, I was wrong